sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ScalaUDF.scala

```diff
@@ -66,980 +85,305 @@ case class ScalaUDF(
+            convertedRow.get(8), convertedRow.get(9), convertedRow.get(10), convertedRow.get(11),
+            convertedRow.get(12), convertedRow.get(13), convertedRow.get(14), convertedRow.get(15),
+            convertedRow.get(16), convertedRow.get(17))
+        }
+
+      case 19 =>
+        val func = function.asInstanceOf[(Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any) => Any]
+        (input: InternalRow) => {
+          val convertedRow: Row = inputEncoder.fromRow(input)
+          func(convertedRow.get(0), convertedRow.get(1), convertedRow.get(2), convertedRow.get(3),
+            convertedRow.get(4), convertedRow.get(5), convertedRow.get(6), convertedRow.get(7),
+            convertedRow.get(8), convertedRow.get(9), convertedRow.get(10), convertedRow.get(11),
+            convertedRow.get(12), convertedRow.get(13), convertedRow.get(14), convertedRow.get(15),
+            convertedRow.get(16), convertedRow.get(17), convertedRow.get(18))
+        }
+
+      case 20 =>
+        val func = function.asInstanceOf[(Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any) => Any]
+        (input: InternalRow) => {
+          val convertedRow: Row = inputEncoder.fromRow(input)
+          func(convertedRow.get(0), convertedRow.get(1), convertedRow.get(2), convertedRow.get(3),
+            convertedRow.get(4), convertedRow.get(5), convertedRow.get(6), convertedRow.get(7),
+            convertedRow.get(8), convertedRow.get(9), convertedRow.get(10), convertedRow.get(11),
+            convertedRow.get(12), convertedRow.get(13), convertedRow.get(14), convertedRow.get(15),
+            convertedRow.get(16), convertedRow.get(17), convertedRow.get(18), convertedRow.get(19))
+        }
+
+      case 21 =>
+        val func = function.asInstanceOf[(Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any) => Any]
+        (input: InternalRow) => {
+          val convertedRow: Row = inputEncoder.fromRow(input)
+          func(convertedRow.get(0), convertedRow.get(1), convertedRow.get(2), convertedRow.get(3),
+            convertedRow.get(4), convertedRow.get(5), convertedRow.get(6), convertedRow.get(7),
+            convertedRow.get(8), convertedRow.get(9), convertedRow.get(10), convertedRow.get(11),
+            convertedRow.get(12), convertedRow.get(13), convertedRow.get(14), convertedRow.get(15),
+            convertedRow.get(16), convertedRow.get(17), convertedRow.get(18), convertedRow.get(19),
+            convertedRow.get(20))
+        }
+
+      case 22 =>
+        val func = function.asInstanceOf[(Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any) => Any]
+        (input: InternalRow) => {
+          val convertedRow: Row = inputEncoder.fromRow(input)
+          func(convertedRow.get(0), convertedRow.get(1), convertedRow.get(2), convertedRow.get(3),
```

Comment:
Since you are breaking the lines manually, it's good to have shorter name instead `convertedRow`